changelogs/unreleased/gh-8861-memtx.sort_threads-option.md

```diff
@@ -0,0 +1,3 @@
+## feature/config
```

Comment:
Needs rebase after PR #8877 and removing of the option from the ignore list in the `test_box_cfg_coverage` test case.

Reply:
Done.